velox/expression/tests/ExprTest.cpp

```diff
@@ -1273,6 +1273,27 @@ TEST_F(ExprTest, switchExpr) {
   assertEqualVectors(expected, result);
 }
 
+TEST_F(ExprTest, switchExprWithNull) {
```

Comment:
I believe you mentioned that this test is for a code path that was broken. Would you update the PR description to explain the problem and the solution?

Reply:
There is no bug here. When I read switchExpr code path, I thought AND condition bits and null bits are incorrect. So I write a test with input having nulls.